low-level/files/opt/seagate/sspl/setup/resource_map/storage.py

```diff
@@ -126,27 +126,25 @@ def get_psu_info(self):
         data = []
         psus = self.get_realstor_encl_data("power-supplies")
         for psu in psus:
-            psu_dict = {
-              "uid": psu.get("durable-id"),
-              "fru": "true",
-              "last_updated": int(time.time()),
-              "health": {
-                "status": psu.get("health", "NA"),
-                "description": psu.get("description"),
-                "recommendation": psu.get("health-recommendation"),
-                "specifics": [
-                  {
+            uid = psu.get("durable-id")
+            status = psu.get("health", "NA")
+            description = psu.get("description")
+            recommendation = psu.get("health-recommendation")
+            psu_dict = self.get_health_template(uid, is_fru=True)
+            psu_dict["last_updated"] = int(time.time())
```

Comment:
Remove this and ensure "last_updated" is set in get_health_template() itself.

Reply:
Apply this wherever applicable.

Reply:
Done.